src/src/lms/lms.service.ts

```diff
@@ -419,5 +379,428 @@ export class LMSService {
 		}
 	}
 
-	//delete
+	//cron issue certificate run every 5 minutes
+	@Cron(CronExpression.EVERY_5_MINUTES)
+	async issueCertificate() {
+		console.log('cron job: issueCertificate started at time ' + new Date());
+		//fetch all test tracking data which has certificate_status null
+		const userForIssueCertificate = await this.fetchTestTrackingData(
```

Comment:
for wch test?

Reply:
for now we are not considering any test here. this cron job will run for any test containing in testTracking data and generate certificate from given criteria